tests/test_polaraveraging.py

```diff
@@ -0,0 +1,234 @@
+                            [[0.00069813170079771690, 0.00106951489719733675], [-1.57079632679489655800, 1.77544123918164542530]],
+                            rtol, err_msg="Failure when lens and a source share an RA")
+    testing.assert_allclose(pa._compute_theta_phi(ra_l, dec_l, ra_s, np.array([51.49, 51.55])),
+                            [[0.00032601941539388962, 0.00106951489719733675], [0.00000000000000000000, 1.77544123918164542530]],
+                            rtol, err_msg="Failure when lens and a source share a DEC")
+    # l/s at same ra AND dec
+    testing.assert_raises(ValueError, pa._compute_theta_phi, ra_l, dec_l, np.array([161.32, 161.34]), np.array([51.49, 51.55]))
+
+    #testing.assert_warns(UserWarning,pa._compute_theta_phi, ra_l, dec_l, np.array([151.32, 161.34]), np.array([41.49, 51.55]),'flat')
+    
+    #testing.assert_allclose(pa._compute_theta_phi(ra_l, dec_l, np.array([161.32, 161.34]), np.array([51.49, 51.55])),
+    #                        [[0.00000000000000000000, 0.00106951489719733675], [0.00000000000000000000, 1.77544123918164542530]],
+    #                        rtol, err_msg="Failure when lens and a source share an RA and a DEC")
+    
+    
+    testing.assert_allclose(pa._compute_theta_phi(0.1, dec_l, np.array([359.9,180.1]), dec_s),
+                            [[2.37312589, 1.95611677], [-2.94182333e-04,  3.14105731e+00]],
+                            rtol, err_msg="Failure when ra_l and ra_s are close but on the two sides of the 0 axis")   
+
+    testing.assert_allclose(pa._compute_theta_phi(0, dec_l, np.array([359.9,180.1]), dec_s),
+                            [[2.37203916, 1.9572035], [-2.94317111e-04,  3.14105761e+00]],
+                            rtol, err_msg="Failure when ra_l and ra_s are separated by pi + epsilon")  
+    
+    testing.assert_allclose(pa._compute_theta_phi(-180, dec_l, np.array([180.1,-90]), dec_s),
+                            [[2.36986569, 0.97805881], [-2.94587036e-04,  3.14052196e+00]],
+                            rtol, err_msg="Failure when ra_l and ra_s are the same but one is defined negative")     
+    
+    testing.assert_allclose(pa._compute_theta_phi(ra_l, 90, ra_s, np.array([51.45,-90])),
+                            [[0.67282443, 3.14159265], [-1.57079633, -1.57079633]],
+                            rtol, err_msg="Failure when dec_l and dec_s are separated by 180 deg")  
+    
+    # ra1, ra2 = .1 and 359.9
+    # testing.assert_allclose(pa._compute_theta_phi(), desired, rtol, err_msg="")
+    # ra1, ra2 = 0, 180.1
+    # testing.assert_allclose(pa._compute_theta_phi(), desired, rtol, err_msg="")
+    # ra1, ra2 = -180, 180
+    # testing.assert_allclose(pa._compute_theta_phi(), desired, rtol, err_msg="")
+    # dec1, dec2 = 90, -90
+    # testing.assert_allclose(pa._compute_theta_phi(), desired, rtol, err_msg="")
+
+
+def test_compute_shear():
+    # the numbers on this test must be revise, output values were extracted from the first run of the code
```

Comment:
do we still need to revise these numbers?

Reply:
No. Lines 180 to 188 can be deleted.